.github/pull_request_template/bugfix.md

```diff
@@ -0,0 +1,26 @@
+# 🐛 Bug Fix
+
+## Checklist
+- [ ] Issue: #___
+- [ ] Branch: `patch-[issue]-[desc]`
+- [ ] Title: `<component>: <message> Fix #<issue>`
+- [ ] Added regression test
+- [ ] Doesn't break existing functionality
+- [ ] Updated docs (if needed)
```

Comment:
Let's standardize this point so that it's the same in all templates. I would suggest using 'Updated documentation' as used in the feature template